gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -123,17 +122,17 @@
         RetryParams.newBuilder()
             .setRetryBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(100L)
+                    .setInitialDelay(Duration.millis(100L))
                     .setDelayMultiplier(1.2)
-                    .setMaxDelayMillis(1000L)
+                    .setMaxDelay(Duration.millis(1000L))
                     .build())
             .setTimeoutBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(300L)
-                    .setDelayMultiplier(1.3)
-                    .setMaxDelayMillis(3000L)
+                    .setInitialDelay(Duration.millis(2000L))
+                    .setDelayMultiplier(1.5)
+                    .setMaxDelay(Duration.millis(30000L))
                     .build())
-            .setTotalTimeout(30000L)
+            .setTotalTimeout(Duration.millis(45000L))
```

Comment:
Using Duration instead of long can avoid potential issues of time units (seconds/milliseconds)